pkg/controller/bundlec_test/finalizer_added_if_not_present_test.go

```diff
@@ -87,8 +87,6 @@ func TestFinalizerAdded(t *testing.T) {
 			updateBundle := bundleUpdate.GetObject().(*smith_v1.Bundle)
 			// Make sure that the "deleteResources" finalizer was added
 			assert.True(t, resources.HasFinalizer(updateBundle, bundlec.FinalizerDeleteResources))
-
-			tc.assertObjectsToBeDeleted(t, m1)
```

Comment:
When finalizer is added, we don't touch `.status` anymore (see above). So no need to test it.